systemd/hitch.conf

```diff
@@ -43,26 +43,29 @@ ocsp-dir = "/var/lib/hitch-ocsp"
 # Verification of OCSP responses
 # ocsp-verify-staple = off
 
-# SSL protocol.
+# List of allowed cipher TLS suites
 #
-# tls = on
-# ssl = off
+# type: string
+ciphersuites = "TLS_AES_128_GCM_SHA256:TLS_AES_256_GCM_SHA384:TLS_CHACHA20_POLY1305_SHA256"
 
-# List of allowed SSL ciphers.
+# List of allowed TLS ciphers.
 #
-# Run openssl ciphers for list of available ciphers.
 # type: string
-ciphers = "EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH"
+ciphers = "ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384"
```

Comment:
are we ok with those regarding "best practices" and strong ciphers?

Reply:
We could add a comment/reference that this matches Intermediate compatibility (https://wiki.mozilla.org/Security/Server_Side_TLS#Intermediate_compatibility_.28recommended.29) with a date reference, and attempt to keep it updated on each release.